orders/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Check if rabbit is up and running before starting the service.
+
+is_ready() {
+    eval "curl -I ${RABBIT_USER}:${RABBIT_PASSWORD} http://rabbit:15672/api/vhosts"
+}
+
+i=0
+while ! is_ready; do
+    i=`expr $i + 1`
+    if [ $i -ge 10 ]; then
+        echo "$(date) - rabbit still not ready, giving up"
+        exit 1
+    fi
+    echo "$(date) - waiting for rabbit to be ready"
+    sleep 3
+done
+
+# Run Migrations
+
+alembic upgrade head
```

Comment:
Probably worth nothing that you *cannot* do backwards migrations with this docker container, and that this is just to get you started.

Alternatively we could remove alembic completely from the example app.